### DISCUSSION TITLE
[BorgBackup] Errors during integrity check

### DISCUSSION THREAD

**Discussion by Roman991** [nextcloud/all-in-one#1505] [Ideas] [answered]
### How to use GitHub
* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Is your feature request related to a problem? Please describe. 

This backup tool is just amazing, but what to do when it fails or the archive gets corrupted and you notice it before the fateful moment?

This is the output of backup check integrity of borgbackup
```shell
Checking the backup integrity...
Index object count mismatch.
committed index: 68256 objects
rebuilt index:   68257 objects
ID: dd259ffc23cbeba2bbee8c88b170e34eeca669b6dd8b7d415c33a2b1e9ecc235 rebuilt index: (37, 516807375)  committed index: <not found>     
Finished full repository check, errors found.
Some errors were found while checking the backup integrity!
```
### Describe the solution you'd like

Recreate a new backup (`check --repair`  might lead to data loss so better not risk).
Idk if add a sheduled `check` would be a good idea? some errors or inconsistencies are better to know before a situation where you have to manage a backup

**Comment by Roman991** [comment, marked as answer]
I just [deleted](https://github.com/nextcloud/all-in-one#delete-backup-archives-manually) all backups and recreated a new one.
No errors so far :)

**Comment by szaimen** [comment]
Looks like a bug in borgbackup unfortunately:
https://github.com/borgbackup/borg/issues/6781
https://github.com/borgbackup/borg/issues/6687
